shunting-yard-emitter/shunting-yard-emitter-kafka/src/test/java/com/hotels/shunting/yard/emitter/kafka/listener/KafkaMetaStoreEventListenerTest.java

```diff
@@ -97,7 +106,8 @@ public void onCreateTable() throws Exception {
     when(serializableEvent.getTableName()).thenReturn(TABLE);
     when(serializableListenerEventFactory.create(event)).thenReturn(serializableEvent);
     listener.onCreateTable(event);
-    verify(executorService).submit(messageTask);
+    verify(executorService).submit(captor.capture());
```

Comment:
Why are you using a `captor` if you are not asserting anything on it?

Reply:
ah got it reading the comments made by Adrian now

Reply:
leftover from last change I made, I'll remove them